src/main/java/org/apache/mesos/scheduler/repair/RepairController.java

```diff
@@ -0,0 +1,37 @@
+package org.apache.mesos.scheduler.repair;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.util.concurrent.atomic.AtomicReference;
+
+/**
+ * Simple endpoint to expose repair state information.
+ * <p>
+ * Renders the stopped tasks and failed tasks as JSON, so that the operator can understand the status of the system.
+ */
+@Path("/v1/repair")
+public class RepairController {
```

Comment:
In dcos-commons these types of classes are called `RepairResource`.  The convention in Kafka of using a `Controller` suffix is outdated.